x-pack/plugins/stack_alerts/server/rule_types/es_query/lib/fetch_search_source_query.ts

```diff
@@ -248,5 +248,5 @@ export function updateFilterReferences(
 export function getSmallerDataViewSpec(
   dataView: DataView
 ): DiscoverAppLocatorParams['dataViewSpec'] {
-  return dataView.toMinimalSpec();
+  return dataView.toMinimalSpec({ keepFieldAttrs: ['customLabel'] });
```

Comment:
Excluding `customDescription` for alert links. `customLabel` is what we allowed to include previously too.